modules/@angular/upgrade/test/upgrade_aot_spec.ts

```diff
@@ -6,14 +6,17 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Injector, Class, Component, OnChanges, OnDestroy, SimpleChanges, EventEmitter,
-        NO_ERRORS_SCHEMA, NgModule, NgModuleRef, OpaqueToken, Testability, destroyPlatform,
-        forwardRef} from '@angular/core';
-import {async, fakeAsync, tick} from '@angular/core/testing';
-import {BrowserModule, platformBrowser, } from '@angular/platform-browser';
-import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
-import {UpgradeModule, ng2ProviderFactory, ng1ServiceProvider, downgradeNg2Component} from '@angular/upgrade';
-import {parseFields} from '@angular/upgrade/src/metadata';
+import {
+  Injector, Class, Component, OnChanges, OnDestroy, SimpleChanges, EventEmitter, NO_ERRORS_SCHEMA,
+  NgModule, NgModuleRef, OpaqueToken, Testability, destroyPlatform, forwardRef
+} from '@angular/core';
+import { async, fakeAsync, tick } from '@angular/core/testing';
+import { BrowserModule, platformBrowser } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import {
+  UpgradeModule, ng2ProviderFactory, ng1ServiceProvider, downgradeNg2Component
+} from '@angular/upgrade';
+import { parseFields } from '@angular/upgrade/src/metadata';
```

Comment:
Is this reformatting from running the code through the formatting engine?
If not, I don't think there is much point in spending time doing this because the engine has its own weird ways of laying things out.